dynawo/sources/Solvers/Common/DYNSolverImpl.cpp

```diff
@@ -138,9 +138,9 @@ Solver::Impl::init(const double& t0, const boost::shared_ptr<Model> & model) {
 
   double *idx = NV_DATA_S(yId_);
   for (int ieq = 0; ieq < model->sizeY(); ++ieq) {
-    idx[ieq] = ONE;
```

Comment:
Maybe it would be clearer to remove the ";" from the macro

Reply:
ok